source/Octo/Diagnostics/LogUtilities.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Octopus.Cli.Commands;
+using Octopus.Cli.Infrastructure;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Octopus.Cli.Diagnostics
+{
+    internal static class LogUtilities
+    {
+        private static readonly Dictionary<string, LogEventLevel> _lookup = new Dictionary<string, LogEventLevel>(StringComparer.OrdinalIgnoreCase)
+        {
+            { "f", LogEventLevel.Fatal },
+            { "fatal", LogEventLevel.Fatal },
+            { "e", LogEventLevel.Error },
+            { "error", LogEventLevel.Error },
+            { "w", LogEventLevel.Warning },
+            { "warning", LogEventLevel.Warning },
+            { "i", LogEventLevel.Information },
+            { "info", LogEventLevel.Information },
+            { "information", LogEventLevel.Information },
+            { "d", LogEventLevel.Debug },
+            { "debug", LogEventLevel.Debug },
+            { "v", LogEventLevel.Verbose },
+            { "verbose", LogEventLevel.Verbose }
+        };
+
+        public static LoggingLevelSwitch LevelSwitch { get; }
+
+        static LogUtilities()
+        {
+            LevelSwitch = new LoggingLevelSwitch(LogEventLevel.Debug);
+        }
+        
+        public static LogEventLevel ParseLogLevel(string value)
+        {
+            if (_lookup.TryGetValue(value, out var level))
+            {
+                return level;
+            }
+            throw new CommandException($"Unrecognized loglevel '{value}'.");
```

Comment:
We try to provide valid options when displaying an error. So this code would print:

> Unrecognized loglevel 'blah'.

Ideally it should print something like:

> Unrecognized loglevel 'blah'. Valid options are fatal, error, warning, information, debug or verbose.